src/style.css

```diff
@@ -1,3 +1,106 @@
-body {
+.roboto-regular {
+  font-family: Roboto, serif;
+  font-style: normal;
+}
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Roboto, serif;
+  font-weight: 500;
+  font-style: normal;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  background: #fff;
+  padding: 0 20px;
+}
+
+.moyoLogo {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 40px;
+  height: 40px;
+  margin-left: 30px;
+}
+
+.nav-links {
+  transform: translateX(-30px);
+  height: 60px;
+  margin-left: auto;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav-links ul {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  list-style: none;
+}
+
+.nav-links li + li {
+  margin-left: 20px;
+  line-height: 60px;
+}
+
+.nav-item {
+  font-size: 12px;
+  text-transform: uppercase;
+  color: #000;
+  text-decoration: none;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+}
+
+.nav-item:hover {
+  color: rgba(0, 172, 220, 1);
+}
+
+.is-active {
+  display: flex;
+  color: rgba(0, 172, 220, 1);
+  align-items: center;
+  justify-content: center;
+  background: white;
+  line-height: 60px;
+}
+
+a.is-active {
+  display: flex;
+  align-items: center;
+  background-color: #fff;
+}
```

Comment:
The styles for `.is-active` and `a.is-active` are quite similar. Consider consolidating these styles to avoid redundancy and improve maintainability. If there are specific styles that only apply to `a.is-active`, separate them clearly.